code/modules/reagents/reagent_containers.dm

```diff
@@ -11,6 +11,7 @@
 	var/spawned_disease = null
 	var/disease_amount = 20
 	var/spillable = 0
+	var/you_drink_from_this = FALSE

```

Comment:
This is disgusting.

Reply:
I'm letting my code speak for itself okay